### PR TITLE
VBM: fix white space under the header

### PR DESCRIPTION
**Pull request by Val-M9**

### CODE REVIEW

File: packages/mobile/src/screens/product-info/product-info.tsx [hunk 1]
```diff
@@ -81,10 +81,10 @@ const ProductInfo: FC = () => {
   const isAuction = type == ProductType.AUCTION;
 
   return (
-    <ScreenWrapper>
+    <ScreenWrapper style={{ marginTop: -25 }}>
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/product-info/product-info.tsx:84]: Emm...

Comment (Val-M9) [reply packages/mobile/src/screens/product-info/product-info.tsx:84]: I assume you mean to place style to styles component

File: packages/mobile/src/screens/product-info/product-info.tsx [hunk 2]
```diff
@@ -67,7 +67,11 @@ const ProductInfo: FC = () => {
   }, [id, user, dispatch]);
 
   if (!product) {
-    return <Spinner />;
+    return (
+      <View style={[globalStyles.flex1, globalStyles.justifyContentCenter]}>
+        <Spinner />
+      </View>
+    );
```
Comment (VladimirBludov) [inline packages/mobile/src/screens/product-info/product-info.tsx:74]: You can just use - `return <Spinner isOverflow={true} />`

**Review by VladimirBludov — CHANGES_REQUESTED** [review]

File: packages/mobile/src/screens/product-info/product-info.tsx (hunk 2 above)
Comment (Val-M9) [reply packages/mobile/src/screens/product-info/product-info.tsx:74]: fixed

File: packages/mobile/src/screens/product-info/components/price-block/lot-price-block.tsx [hunk 3]
```diff
@@ -177,7 +177,11 @@ const LotPriceBlock: FC<LotPriceBlockProps> = ({
               globalStyles.alignItemsCenter,
             ]}
           >
-            {isLoading && <Spinner />}
+            {isLoading && (
+              <View>
+                <Spinner isOverflow={true} />
+              </View>
+            )}
```
[the remark this answers is not in the document]
Comment (Val-M9) [reply packages/mobile/src/screens/product-info/components/price-block/lot-price-block.tsx:184]: you right, this way looks prettier

**Review by VladimirBludov — APPROVED** [review]

**Review by natata7 — APPROVED** [review]

**Review by mazurGit — APPROVED** [review]

**Pull request merged**